components/MenuEditorials/index.js

```diff
@@ -16,7 +16,7 @@ const titleDefaultProps = {
   mb: 3,
 };
 
-const MenuEditorials = ({amp, color, items, title, titleProps, groupTitleProps, px, py, removeBorders}) => {
+const MenuEditorials = ({amp, color, items, title, titleProps, groupSubItemProps, groupTitleProps, px, py, removeBorders}) => {
```

Comment:
Ajuste para a sugestão anterior

```suggestion
const MenuEditorials = ({amp, color, items, title, titleProps, groupSubItemProps, groupTitleProps, px, py, removeBorders, menuItemProps}) => {
```